tests/features/bootstrap/FeatureContext.php

```diff
@@ -29,6 +35,229 @@ public function assertAccessDenied() {
   }
 
   /**
+   * Assert that certain fields are present on the page.
+   *
+   * @param string $fields
+   *    Fields.
+   *
+   * @throws \Exception
```

Comment:
There are a few reason behind this. After taking a small discussion with @pfrenssen and @sandervd too:
- we are not going to catch the exception, we just want to stop the test. So we don't need to use anything more than `\Exception` for steps. Steps implemented by Behat are throwing default exceptions too.
- `ElementNotFoundException` is coming from Mint, our context is related to Behat. Behat suggests Mink package, but doesn't require it. So we shouldn't throw those exceptions here.